docs/content/Authentication-&-Authorization.md

```diff
@@ -144,19 +149,19 @@ can then use it to query the Cube.js API.
 ## Security Context
 
 A "security context" is a verified set of claims about the current user that the
-cube.js server can use to ensure that users only have access to the data that
+Cube.js server can use to ensure that users only have access to the data that
```

Comment:
@hassankhan Can we have some linter rules for frequently used names such as "Cube.js" (vs "cube.js", "CubeJS", etc.)?

Reply:
@igorlukanin Definitely on the agenda 😄